### ISSUE TITLE
Suggestion: Mining Job: Never mine out any tile with thick roof.

### ISSUE THREAD

**Issue by maarxx** [FluffierThanThou/ColonyManager#183]
**Problem description**

I'm on an open field map, and I want to mine some loose Steel / Machinery / etc, but I don't want to expose myself to Insect events. Right now, there is no exposed thick roof anywhere, and I want to keep it that way.

My understanding is none of the existing options for roof support will cover this.

**Describe the solution**

New Health & Safety option: Never mine out any tile with thick roof.

**Comment by maarxx** [comment]
I might look into adding this myself but wanted to post it up here early to stew and get feedback.

**Comment by shiribailem** [comment]
I'm pretty sure it's not there because, barring mods that enable it, you can't dig out thick roofs.

**Comment by maarxx** [comment]
No, I mean, option would say "Never mine out any tile [covered with] with a thick roof".

For cases where you build outdoors on an open map, there are only a few large mountains, and currently Insects are not a possibility, and you want to keep it that way. When there is plenty other mineral to mine that isn't covered with a thick roof.
